### PR TITLE
fix: VA County Names

### PR DESCRIPTION
**Pull request by aed3**
The names of some Virginia counties end with "city" in the data base I pulled from while they are "County" in the geoJSON file

### CODE REVIEW

File: src/events/crawler/scrapers/USA/VA/index.js [hunk 1]
```diff
@@ -126,25 +125,25 @@ const scraper = {
...
     'Tazewell County',
-    'Virginia Beach city',
+    'Virginia Beach County',
     'Warren County',
     'Washington County',
-    'Waynesboro city',
+    'Waynesboro County',
     'Westmoreland County',
-    'Williamsburg city',
-    'Winchester city',
```
Comment (lazd) [inline src/events/crawler/scrapers/USA/VA/index.js]: Ok, so these are actually counties, not cities? Or are they city-counties (same name for both city and county)?

Comment (aed3) [reply src/events/crawler/scrapers/USA/VA/index.js]: Some of them are city-counties while others are different counties and cities with the same name.

**Comment by lazd** [comment]
@aed3 can you please separate the VA counties from the NE scraper and send the NE scraper in a separate PR?

**Comment by aed3** [comment]
> @aed3 can you please separate the VA counties from the NE scraper and send the NE scraper in a separate PR?

Yes

**Pull request merged**